database/query/providers.sql

```diff
@@ -58,4 +58,4 @@ UPDATE providers
     WHERE id = sqlc.arg('id') AND project_id = sqlc.arg('project_id');
 
 -- name: DeleteProvider :exec
-DELETE FROM providers WHERE id = $1 AND project_id = $2;
\ No newline at end of file
+DELETE FROM providers WHERE id = $1;
```

Comment:
Note that this function was unused before. Given that we already are passing UUID I hope that it should be possible to delete without passing the project as well?

@JAORMX WDYT?

Reply:
also added tests